### ISSUE TITLE
[Bug]: blob spore zombie AI does not turn off when a ghost inhabits it

### ISSUE THREAD

**Issue by kiekatz** [BeeStation/BeeStation-Hornet#11999]
### BYOND Version

515.1614

### Round Date

2024/12/07

### Round ID

50893

### Test merges

[#9124: 'Enable serverside 515.1614' by Crossedfall at commit 1b8f225789](https://www.github.com/beestation/beestation-hornet/pull/9124)
[#10897: '[PORT] Follows up TG's atom_hud system' by EvilDragonfiend at commit 51f8edef4f](https://www.github.com/beestation/beestation-hornet/pull/10897)

### Describe the bug

When I [spawned as a blob zombie], I noticed [the blobzombie AI is still active. resulting in me and IT fighting over the body as IT continues to chase targets or forcefully remain still on a given tile]

### What did you expect?

I expected [the AI to turn off so i dont have to fight it when playing a blob zombie]

### Reproducible steps

1. inhabit a blob zombie who's AI is actively hunting someone

**Comment by Fronsis** [comment]
Yeah, i've pened an issue about his some months ago but it got automarked as stale :skull: very annoying

**Comment by Fronsis** [comment]
Still active
